File: GumiPresso_Server/src/main/java/com/ssafy/through/model/service/ImageService.java
package com.ssafy.through.model.service;

import org.springframework.web.multipart.MultipartFile;

import com.ssafy.through.model.dto.ImageFile;

public interface ImageService {
	int insert(ImageFile imageFile);
	ImageFile select(int id);	
	void deleteFile(String imageFileName);
	void fileUpload(MultipartFile multipartFile, String flag);
}

File: GumiPresso_Admin/app/src/main/java/com/ssafy/gumipresso_admin/model/service/ImageService.kt
package com.ssafy.gumipresso_admin.model.service

import okhttp3.MultipartBody
import okhttp3.RequestBody
import org.w3c.dom.Comment
import retrofit2.Response
import retrofit2.http.*

interface ImageService {
    @GET("/comment/{product_id}")
    suspend fun getComments(@Path("product_id") productId: Int): Response<MutableList<Comment>>

    @Multipart
    @POST("admin/product/insert")
    suspend fun insertProduct(@Part imageFile: MultipartBody.Part, @PartMap map: Map<String, RequestBody>): Response<Void>

}